Software/app/src/main/java/org/foi/air/smartcharger/MainActivity.kt

```diff
@@ -2,10 +2,37 @@ package org.foi.air.smartcharger
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.widget.Button
+import android.widget.TextView
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        val button = findViewById<Button>(R.id.button1)
```

Comment:
I don't think button1 is a good name for a button, please change the name so it's use is described.